http/grpc/src/test/java/io/quarkus/ts/http/grpc/GrpcMtlsEncryptedPemTlsRegistryIT.java

```diff
@@ -34,7 +37,7 @@ public void testMutualTlsCommunicationWithHelloService() {
         try (var channel = app.securedGrpcChannel()) {
             HelloRequest request = HelloRequest.newBuilder().setName(CLIENT_CN_NAME).build();
             HelloReply response = GreeterGrpc.newBlockingStub(channel).sayHello(request);
-            assertEquals("Hello " + CLIENT_CN_NAME, response.getMessage());
+            Assertions.assertEquals("Hello " + CLIENT_CN_NAME, response.getMessage());
```

Comment:
any reason for this change?

Reply:
I realized we usually use org.junit.jupiter.api.Assertions; instead of org.junit.Assert.assertEquals; so it's to homogenize the assertions.